agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/BaseStatsbeat.java

```diff
@@ -39,7 +39,7 @@ protected StatsbeatTelemetryBuilder createStatsbeatTelemetry(
 
     StatsbeatTelemetryBuilder telemetryBuilder = StatsbeatTelemetryBuilder.create(name, value);
 
-    telemetryBuilder.setInstrumentationKey(telemetryClient.getStatsbeatInstrumentationKey());
+    telemetryBuilder.setInstrumentationKey(telemetryClient.getInstrumentationKey());
```

Comment:
this will use the customer's ikey.. telemetryClient.getStatsbeatConnectionString().getInstrumentationKey()?

Reply:
oh good catch 👍